org.mosh.mosh.appdata.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>org.mosh.mosh</id>
+  <name>Mobile Shell</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <provides><binary>mosh</binary></provides>
+  <description>
+    <p>Mosh is a remote terminal application that supports intermittent
+    connectivity, allows roaming, and provides speculative local echo
+    and line editing of user keystrokes.</p>
+
+    <p>It aims to support the typical interactive uses of SSH, plus:</p>
+
+    <ul>
+      <li>Mosh keeps the session alive if the client goes to sleep and
+      wakes up later, or temporarily loses its Internet connection.</li>
+
+      <li>Mosh allows the client and server to "roam" and change IP
+      addresses, while keeping the connection alive. Unlike SSH, Mosh
+      can be used while switching between Wi-Fi networks or from Wi-Fi
+      to cellular data to wired Ethernet.</li>
+
+      <li>The Mosh client runs a predictive model of the server's behavior
+      in the background and tries to guess intelligently how each
+      keystroke will affect the screen state. When it is confident in
+      its predictions, it will show them to the user while waiting for
+      confirmation from the server. Most typing and uses of the left-
+      and right-arrow keys can be echoed immediately.</li>
+    </ul>
+
+    <p>As a result, Mosh is usable on high-latency links, e.g. on a
+    cellular data connection or spotty Wi-Fi. In distinction from
+    previous attempts at local echo modes in other protocols, Mosh
+    works properly with full-screen applications such as emacs, vi,
+    alpine, and irssi, and automatically recovers from occasional
+    prediction errors within an RTT. On high-latency links, Mosh
+    underlines its predictions while they are outstanding and removes
+    the underline when they are confirmed by the server.</p>
+
+    <p>Mosh does not support X forwarding or the non-interactive uses of SSH,
+    including port forwarding.</p>
+  </description>
+  <developer_name>Keith Winstein, John Hood, et al.</developer_name>
+  <releases>
+    <release date="2017-07-22" version="1.3.2">
+      <p>Platform support:</p>
```

Comment:
Did you manually reformat these from [the release notes](https://mailman.mit.edu/pipermail/mosh-users/2017-July/000357.html)?

Reply:
Yes.